From c51a67772196e7040e0d965da2c5ce4dc502d3a3 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Mon, 19 Feb 2018 21:18:42 +0900
Subject: [PATCH 663/909] arm64: dts: r8a77990-ebisu: Enable ADSP

commit 21593e5ee24531cfed26d1243259d9c7c0d65ce6 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch enables Audio DSP (ADSP) for the Ebisu board on the R8A77990
SoC.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts b/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
index d30a6497e371..ef343ffd058b 100644
--- a/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
@@ -342,6 +342,11 @@
 	};
 };
 
+&adsp {
+	status = "okay";
+	memory-region = <&adsp_reserved>;
+};
+
 &audio_clk_a {
 	clock-frequency = <22579200>;
 };
-- 
2.17.1

